src/edu/ucsd/library/xdre/statistic/analyzer/LogAnalyzer.java

```diff
@@ -90,11 +90,17 @@ public void analyze(File logFile) throws IOException{
 					if (line.indexOf("\"-\"", spIdx) > 0 || line.indexOf("archive.org_bot", spIdx) > 0)
 						continue;
 
-					// ignore hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
-					if (line.indexOf("http://library.ucsd.edu/speccoll/", spIdx) > 0)
-						continue;
-					
 					uri = getUri(line);
+					// Hits from MSCL collection pages: http://library.ucsd.edu/speccoll/
```

Comment:
@mcritchlow I've extracted the method out and refactored the codes (see commit https://github.com/ucsdlib/damsmanager/pull/131/commits/47ec4efbea0efde03e5fafca23dff3547537c8c6) to include anything from SC&A exhibits in DAMS stats with _2.jpg being counted as object view.